lib/tag-list.jsx

```diff
@@ -64,7 +64,7 @@ export class TagList extends Component {
       <div className={classes}>
         <div className="tag-list-title">
           <h2 className="panel-title theme-color-fg-dim">Tags</h2>
-          {tags.length && (
+          {0 !== tags.length && (
```

Comment:
good catch! either this or maybe `tags.length > 0 &&` might be more common. this is good either way

Reply:
cool, I'll correct that.